addon/components/o-s-s/smart/tag.ts

```diff
@@ -0,0 +1,68 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { runSmartGradientAnimation } from '@upfluence/oss-components/utils/run-smart-gradient-animation';
+
+export type TagType = 'keyword' | 'hashtag' | 'mention';
+
+const TypeColorMatch: any = {
+  hashtag: 'cyan',
+  mention: 'violet',
+  keyword: 'yellow'
+};
+
+const TypeIconMatch: any = {
+  hashtag: 'fa-hashtag',
+  mention: 'fa-at',
+  keyword: null
+};
```

Comment:
how about merging these two into a single definition and adding some typing ?
( also will prevent using `any` for no reason ^^ )

```typescript
export type TagType = 'keyword' | 'hashtag' | 'mention';
export type TagTypeDefinition = {
  icon?: string;
  color: string;
};

const TypeDefinition: Record<TagType, TagTypeDefinition> = {
  hashtag: {
    icon: 'fa-hashtag',
    color: 'cyan'
  },
  mention: {
    icon: 'fa-at',
    color: 'violet'
  },
  keyword: {
    color: 'yellow'
  }
};
```